agent/http-server.c

```diff
@@ -23,6 +23,7 @@
 
 #include <json-glib/json-glib.h>
 
+#include <hwangsae/recorder.h>
```

Comment:
What does `HwangsaeHttpServer` need recorder for?

Reply:
With the approach I used, it needs it to access `hwangsae_common_get_local_ip` using the same include file. Probably you prefer to use `hwangsae/common.h>`, as I explained, I try to keep one include file per type of module. Again, no problem in change it.